src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -50,6 +50,11 @@ public function fire()
 	 */
 	protected function getStub()
 	{
+		if ( ! empty($this->laravel['config']['console.stubs.model.default']))
```

Comment:
The problem with `$config->get()` resides in the fact that it doesn't fallback to default if config option is set to null or empty string.

Do you think it's better to use `$config->get()` and several commented lines here https://github.com/Arrilot/laravel/blob/custom_stubs/config/console.php?